package.json

```diff
@@ -92,8 +92,9 @@
         "react-select": "^2.0.0"
     },
     "description": "This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).",
-    "main": "dist/lib.js",
-    "module": "dist/lib.js",
+    "main": "dist/cjs/lib.js",
+    "module": "dist/esm/lib.js",
```

Comment:
I had to read up real quick on the differences between `main` and `module`.  I mostly found [this](https://github.com/dherman/defense-of-dot-js/blob/master/proposal.md) the most useful, but if you happen to still have other articles that you found helpful in figuring out the changes required for this PR, just for my own education, that'd be great!  Not necessary though, obviously.

Reply:
heh, to be completely honest, I largely used the example package.json from https://github.com/rollup/rollup-starter-lib/blob/master/package.json (originally, I was looking into using rollup.js) plus looking around at what webpack expected (they mention using the `module` on one of the pages about tree-shaking, indicating that it should be a version with ES6 module syntax)